### PR TITLE
Libs / PHPCS: fix linting errors, part 2

### PR DESCRIPTION
**Pull request by jeherve**
See #21949

#### Changes proposed in this Pull Request:

* Fix linting errors in lib directory, part 2.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Is CI happy?

### CODE REVIEW

File: projects/plugins/jetpack/_inc/lib/tonesque.php [hunk 1]
```diff
@@ -137,49 +166,56 @@ function grab_points( $type = 'index' ) {
...
+			$r[]   = $index['red'];
+			$g[]   = $index['green'];
+			$b[]   = $index['blue'];
 
-			$red = round( array_sum( $r ) / 5 );
+			$red   = round( array_sum( $r ) / 5 );
 			$green = round( array_sum( $g ) / 5 );
-			$blue = round( array_sum( $b ) / 5 );
+			$blue  = round( array_sum( $b ) / 5 );
 		}
```
Comment (anomiex) [inline projects/plugins/jetpack/_inc/lib/tonesque.php:216]: WTF?
```suggestion
		foreach ( $rgb as $color ) {
			$index = imagecolorsforindex( $img, $color );
			$r[]   = $index['red'];
			$g[]   = $index['green'];
			$b[]   = $index['blue'];
		}

		$red   = round( array_sum( $r ) / 5 );
		$green = round( array_sum( $g ) / 5 );
		$blue  = round( array_sum( $b ) / 5 );
```

**Review by anomiex — COMMENTED** [review]
Lots of WTFs in that color code. Feel free to ignore any that are just pointing out how we could simplify it, and the two griping about how toHsvFloat and toHsvInt don't even make sense. 😀

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 2]
```diff
@@ -56,122 +65,145 @@ public function __construct( $color = null, $type = 'hex' ) {
...
-		if ( $intValue < 0 || $intValue > 16777215 ) {
-			throw new RangeException( $hexValue . " out of valid color code range" );
+		if ( $int_value < 0 || $int_value > 16777215 ) {
+			throw new RangeException( $hex_value . ' out of valid color code range' );
 		}
 
-		$this->color = $intValue;
+		$this->color = $int_value;
 
 		return $this;
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: 68d1cd38f9557b1f8c30128260e71d7f65e14d5e

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 3]
```diff
@@ -56,122 +65,145 @@ public function __construct( $color = null, $type = 'hex' ) {
...
+			$short     = $hex_value;
+			$i         = 0;
+			$hex_value = '';
 			while ( $i < 3 ) {
-				$chunk = substr($short, $i, 1 );
-				$hexValue .= $chunk . $chunk;
+				$chunk      = substr( $short, $i, 1 );
+				$hex_value .= $chunk . $chunk;
 				$i++;
 			}
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: c4f617403c2ec2e34a0de467286593b8e64a49a2

File: projects/plugins/jetpack/_inc/lib/tonesque.php (hunk 1 above)
Comment (jeherve) [reply projects/plugins/jetpack/_inc/lib/tonesque.php]: ea37bdca0dac5a9a7d8ac828548a36b71e6f567f

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 4]
```diff
@@ -326,19 +353,24 @@ public function toHsvInt()
...
-		}
-		else {
+		$r                /= 255;
+		$g                /= 255;
+		$b                /= 255;
+		$max               = max( $r, $g, $b );
+		$min               = min( $r, $g, $b );
+		$l                 = ( $max + $min ) / 2;
+		$s                 = $l;
+		$h                 = $s;
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: 5ca56c335d64304297dee3ba653e68feddf663a6

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 5]
```diff
@@ -56,122 +65,145 @@ public function __construct( $color = null, $type = 'hex' ) {
 	/**
 	 * Init color from hex value
 	 *
-	 * @param string $hexValue
+	 * @param string $hex_value Color hex value.
 	 *
 	 * @return Jetpack_Color
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/_inc/lib/class.color.php]: 71f354b7694d7b158c39d5ddc9ad89bbad491a42

File: projects/plugins/jetpack/_inc/lib/class.color.php [hunk 6]
```diff
@@ -828,56 +811,66 @@ public function toGrayscale() {
...
 	public function getTetrad( $step = 1 ) {
 		$incr = $step * 60;
 		return $this->incrementHue( $incr );
 	}
 
 	/**
-	 * Transform -- Get hue-incremented color, 120 steps at a time.
+	 * Transform -- To a triadic color.
+	 *
+	 * A triadic color scheme uses three colors evenly spaced (180°) around the color wheel.
```
Comment (anomiex) [inline projects/plugins/jetpack/_inc/lib/class.color.php:868]: Looks like I had a typo in my suggestion :(
```suggestion
	 * A triadic color scheme uses three colors evenly spaced (120°) around the color wheel.
```

**Review by anomiex — APPROVED** [review]

**Pull request merged**